### ISSUE TITLE
Possible to convert this to eu4?

### ISSUE THREAD

**Issue by Epicity** [TKasekamp/VickyWarAnalyzer#3]
Is it possible to change this for eu4? The save files in eu4 about the same info as vicky 2

**Comment by TKasekamp** [comment]
True, the save files are similar. But since I don't own EU4, I can't go to the forums and check if someone has already made an analyzer that shows this info. I'd rather not spend doing something that someone else has already done.
